### PR TITLE
CSI Unity XT docs update

### PR DESCRIPTION
**Pull request by panigs7**
# Description
Documentation update for CSI Unity XT which includes:
- Update of support matrix
- Update of CSI Unity version

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/583 |

# Checklist:

- [ ] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/installation/helm/unity.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ Install CSI Driver for Unity XT using this procedure.
 
 *Before you begin*
 
- * You must have the downloaded files, including the Helm chart from the source [git repository](https://github.com/dell/csi-unity) with the command ```git clone -b v2.5.0 https://github.com/dell/csi-unity.git```, as a pre-requisite for running this procedure.
+ * You must have the downloaded files, including the Helm chart from the source [git repository](https://github.com/dell/csi-unity) with the command ```git clone -b v2.6.0 https://github.com/dell/csi-unity.git```, as a pre-requisite for running this procedure.
```
Comment (rsedlock1958) [inline content/docs/csidriver/installation/helm/unity.md:91]: Move "As a prerequisite to running this procedure," to the front of this sentence.

File: content/docs/csidriver/installation/helm/unity.md [hunk 2]
```diff
@@ -101,8 +101,8 @@ Procedure
     **Note**: 
       * ArrayId corresponds to the serial number of Unity XT array.
       * Unity XT Array username must have role as Storage Administrator to be able to perform CRUD operations.
-      * If the user is using complex K8s version like "v1.21.3-mirantis-1", use below kubeVersion check in helm/csi-unity/Chart.yaml file.
-            kubeVersion: ">= 1.21.0-0 < 1.26.0-0"
+      * If the user is using complex K8s version like "v1.24.6-mirantis-1", use below kubeVersion check in helm/csi-unity/Chart.yaml file.
```
Comment (rsedlock1958) [inline content/docs/csidriver/installation/helm/unity.md:104]: Insert "a" after "using". Replace "below" with "this".

File: content/docs/csidriver/installation/helm/unity.md (hunk 1 above)
Comment (panigs7) [reply content/docs/csidriver/installation/helm/unity.md]: Done

File: content/docs/csidriver/installation/helm/unity.md [hunk 3]
```diff
@@ -101,8 +101,8 @@ Procedure
     **Note**: 
       * ArrayId corresponds to the serial number of Unity XT array.
       * Unity XT Array username must have role as Storage Administrator to be able to perform CRUD operations.
-      * If the user is using complex K8s version like "v1.21.3-mirantis-1", use below kubeVersion check in helm/csi-unity/Chart.yaml file.
-            kubeVersion: ">= 1.21.0-0 < 1.26.0-0"
+      * If the user is using a complex K8s version like "v1.24.6-mirantis-1", use this kubeVersion check in helm/csi-unity/Chart.yaml file.
+            kubeVersion: ">= 1.24.0-0 < 1.26.0-0"
```
Comment (rajkumar-palani) [inline content/docs/csidriver/installation/helm/unity.md:105]: max version should be <1.27.0. please check it once.

File: content/docs/csidriver/troubleshooting/unity.md [hunk 4]
```diff
@@ -12,6 +12,6 @@ description: Troubleshooting Unity XT Driver
...
+| Driver install or upgrade fails because of an incompatible Kubernetes version, even though the version seems to be within the range of compatibility. For example: `Error: UPGRADE FAILED: chart requires kubeVersion: >= 1.24.0 < 1.26.0 which is incompatible with Kubernetes 1.24.6-mirantis-1` | If you are using an extended Kubernetes version, please see the helm Chart at `helm/csi-unity/Chart.yaml` and use the alternate `kubeVersion` check that is provided in the comments. *Please note* that this is not meant to be used to enable the use of pre-release alpha and beta versions, which is not supported. |
```
Comment (rajkumar-palani) [inline content/docs/csidriver/troubleshooting/unity.md:15]: make sure on the K8s Max Version (<1.27.0)?

**Review by rajkumar-palani — CHANGES_REQUESTED** [review]
can you please check the review comments and update?

File: content/docs/csidriver/installation/helm/unity.md (hunk 3 above)
Comment (panigs7) [reply content/docs/csidriver/installation/helm/unity.md]: Done

File: content/docs/csidriver/troubleshooting/unity.md (hunk 4 above)
Comment (panigs7) [reply content/docs/csidriver/troubleshooting/unity.md]: Done

**Review by rajendraindukuri — APPROVED** [review]

**Review by rajkumar-palani — DISMISSED** [review]
lgtm

File: content/docs/csidriver/_index.md [hunk 5]
```diff
@@ -16,13 +16,13 @@ The CSI Drivers by Dell implement an interface between [CSI](https://kubernetes-
...
-| Kubernetes    | 1.24, 1.25, 1.26 | 1.23, 1.24, 1.25    | 1.23, 1.24, 1.25 | 1.23, 1.24, 1.25  | 1.24, 1.25, 1.26 |
+| Kubernetes    | 1.24, 1.25, 1.26 | 1.23, 1.24, 1.25    | 1.24, 1.25, 1.26 | 1.23, 1.24, 1.25  | 1.24, 1.25, 1.26 |
 | RHEL          |     7.x,8.x      |     7.x,8.x         |     7.x,8.x      |     7.x,8.x       |     7.x,8.x      |
 | Ubuntu        |       20.04      |       20.04         |  18.04, 20.04    | 18.04, 20.04      |        20.04     |
 | CentOS        |     7.8, 7.9     |      7.8, 7.9       |     7.8, 7.9     |      7.8, 7.9     |     7.8, 7.9     |
-| SLES          |        15SP4     |        15SP4        |       15SP3      |         15SP3     |       15SP3      |
+| SLES          |        15SP4     |        15SP4        |       15SP4      |         15SP3     |       15SP3      |
 | Red Hat OpenShift | 4.10, 4.10 EUS, 4.11 | 4.10, 4.10 EUS, 4.11 | 4.10, 4.10 EUS, 4.11 | 4.10, 4.10 EUS, 4.11 | 4.10, 4.10 EUS, 4.11 |
-| Mirantis Kubernetes Engine | 3.5.x |      3.6.0        |       3.5.x      |        3.5.x      |        3.6.x     |
+| Mirantis Kubernetes Engine | 3.5.x |      3.6.0        |       3.6.0      |        3.5.x      |        3.6.x     |
```
Comment (rensyct) [inline content/docs/csidriver/_index.md:25]: Should it be 3.6.x?

File: content/docs/csidriver/_index.md [hunk 6]
```diff
@@ -16,13 +16,13 @@ The CSI Drivers by Dell implement an interface between [CSI](https://kubernetes-
...
 | Google Anthos |        1.12       |          1.12        |        no        |         1.12       |        1.12       |
 | VMware Tanzu  |        no        |          no         |        NFS       |         NFS       |      NFS,iSCSI         |
 | Rancher Kubernetes Engine | yes  |          1.4.1        |        yes       |         yes       |      yes         |
```
Comment (rensyct) [inline content/docs/csidriver/_index.md:28]: Should we update the version for RKE too?

File: content/docs/csidriver/_index.md (hunk 5 above)
Comment (panigs7) [reply content/docs/csidriver/_index.md]: Done

File: content/docs/csidriver/_index.md (hunk 6 above)
Comment (panigs7) [reply content/docs/csidriver/_index.md]: Done

File: content/docs/csidriver/release/unity.md [hunk 7]
```diff
@@ -3,12 +3,13 @@ title: Unity XT
...
 
-## Release Notes - CSI Unity XT v2.5.0
+## Release Notes - CSI Unity XT v2.6.0
 
 ### New Features/Changes
 
-- [Added support to Kubernetes 1.25](https://github.com/dell/csm/issues/478)
-- [Added support for OpenShift 4.11](https://github.com/dell/csm/issues/480)
+- [Added support to Kubernetes 1.26](https://github.com/dell/csm/issues/597)
+- Added support for MKE 3.6
```
Comment (rensyct) [inline content/docs/csidriver/release/unity.md:11]: Please mention the Github issue for MKE and RKE

**Review by karthikk92 — APPROVED** [review]

**Review by rensyct — APPROVED** [review]

**Review by shanmydell — APPROVED** [review]

**Pull request merged**